Neo4j.Driver/Neo4j.Driver/V1/Config.cs

```diff
@@ -77,6 +77,11 @@ public class Config
         [System.Obsolete("Do not set idle connection size to infinite.")]
         public const int InfiniteMaxIdleSessionPoolSize = -1;
 
+        /// <summary>
+        /// This const sets the number of connections in connection pool <see cref="MaxConnectionPoolSize"/> to be inifinite,
+        /// </summary>
+        public const int InfiniteMaxConnectionPoolSize = -1;
```

Comment:
Maybe we can just get away with documenting that `-1` means infinite? Otherwise it is strange that we have constant only for this setting and not for `ConnectionAcquisitionTimeout` or `MaxConnectionPoolSize`